hangman.js

```diff
@@ -0,0 +1,147 @@
+
```

Comment:
Very impressive work with game visuals, theme and code. Here are few feedbacks: 
1. Glitch with infinite window.alert when user leaves input box blank. 
2. Some DOM manipulation methods can be saved in smaller functions which I see some instances (win, lose, and restart functions). This will make your code more readable and DRY. Try to be deliberate about making more functions in future projects. 
3. Good background with gif links. I enjoyed the space theme. 

Other than those feedbacks, keep up the good work and looking forward to see your future.